turbopy/core.py

```diff
@@ -599,8 +599,17 @@ def __init__(self, grid_data: dict):
         self.cell_widths = (self.r[1:] - self.r[:-1])
         # This will give a divide-by-zero warning.
         # I'm ok with that for now.
-        self.r_inv = 1 / self.r
-        self.r_inv[0] = 0
+        self.coordinate_system = "cartesian"
+        if "coordinate_system" in grid_data:
+            self.coordinate_system = grid_data["coordinate_system"]
+        self.r_inv = np.zeros_like(self.r)
+        self.r_inv[1:] = 1 / self.r[1:]
```

Comment:
I dealt with this a different way. Please merge new changes in `main` into your dev branch.